Documentation/platforms/xtensa/esp32/boards/esp32-devkitc/index.rst

```diff
@@ -329,6 +329,17 @@ was successful by running ``cxxtest``::
     Invalid file! /invalid
     File /proc/version exists!
 
+dac
+---
+
+This configuration enables DAC and registers a `DAC example application <https://github.com/apache/nuttx-apps/tree/master/examples/dac>`_.
+
+.. note:: The DAC module is hard-wired to pins 25 (channel 0) and 26
+  (channel 1). The default device name is ``/dev/dac0`` and can be changed in
+  the config menu.
+
```

Comment:
Suggestion: include the command line command usage

Reply:
Done